ros2doctor/ros2doctor/api/network.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import platform
+import sys
+
+import ifcfg
+
+from ros2doctor.api.format import print_term
+
+
+def _is_unix_like_platform():
+    """Return True if conforms to UNIX/POSIX-style APIs."""
+    return platform.system() in ['Linux', 'FreeBSD', 'Darwin']
+
+
+def check_sys_ips():
```

Comment:
This function name doesn't seem very descriptive. It's a little more confusing in the context of `ros2doctor` since it's not one of the `ros2doctor.checks`.  I don't have an alternative though.

Reply:
changing to `check_network_config_helper`